### PR TITLE
docs: new repo

### PR DESCRIPTION
**Pull request by Real-MullaC**
### **Make docs URL first like sent in dms!**

https://github.com/Chris-Titus-Docs/linutil-docs - New docs repo

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [x] Documentation update
- [ ] Refactoring
- [ ] Hotfix
- [ ] Security patch
- [ ] UI/UX improvement

## Description

New docs layout and repo

## Testing

All works locally on new repo but none needed here.

## Impact

None

## Issues / other PRs related

- Resolves no issues

## Additional Information

No

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

**Comment by jeevithakannan2** [comment]
@Real-MullaC The preview path in the workflow needs to be changes also

File: README.md [hunk 1]
```diff
@@ -86,6 +86,8 @@ We welcome contributions from the community! Before you start, please review our
 
 [Official LinUtil Roadmap](https://christitustech.github.io/linutil/roadmap)
 
+Docs is now [here](https://github.com/Chris-Titus-Docs/linutil-docs)
```
Comment (adamperkowski) [inline README.md:89]:
```suggestion
Docs are now [here](https://github.com/Chris-Titus-Docs/linutil-docs)
```

**Comment by adamperkowski** [comment]
@Real-MullaC You also need to create a workflow to automatically update `userguide` in the other repo. `cargo xtask docgen`

**Comment by Real-MullaC** [comment]
> @Real-MullaC The preview path in the workflow needs to be updated also

What workflow?

**Comment by adamperkowski** [comment]
> > @Real-MullaC The preview path in the workflow needs to be updated also
> 
> What workflow?

There is none yet. You have to make one. When someone edits `tab_data` it needs to run `cargo xtask docgen` and commit the changes to the other repos

**Comment by Real-MullaC** [comment]
> > > @Real-MullaC The preview path in the workflow needs to be updated also
> > 
> > 
> > What workflow?
> 
> There is none yet. You have to make one. When someone edits `tab_data` it needs to run `cargo xtask docgen` and commit the changes to the other repos

Done both issues

**Review by adamperkowski — APPROVED** [review]
@ChrisTitusTech don't forget the workflow

**Review by ChrisTitusTech — APPROVED** [review]

**Review by ChrisTitusTech — APPROVED** [review]

**Comment by solomoncyj** [comment]
can someone explain the rational for the favicon removal? it is used as the desktop icon for the application @Real-MullaC

**Comment by Real-MullaC** [comment]
> can someone explain the rational for the favicon removal? it is used as the desktop icon for the application for some distrubutions @Real-MullaC 

It will need adding back then. As we moved docs to a new repo.

**Comment by solomoncyj** [comment]
Maybe we should submodule the docs repo into here and delete the repo submodule in the docs

**Pull request merged**